app/src/main/java/com/mybus/activity/SearchActivity.java

```diff
@@ -45,11 +47,11 @@
  */
 public class SearchActivity extends AppCompatActivity implements OnAddressGeocodingCompleteCallback, HistoryItemSelectedListener, FavoriteItemSelectedListener, OnLocationGeocodingCompleteCallback {
 
-    public static final String SEARCH_TITLE_EXTRA = "SEARCH_TITLE_EXTRA";
     public static final String SEARCH_TYPE_EXTRA = "SEARCH_TYPE_EXTRA";
 
     public static final String RESULT_STREET_EXTRA = "RESULT_STREET_EXTRA";
     public static final String RESULT_LATLNG_EXTRA = "RESULT_LATLNG_EXTRA";
+    public static final String SEARCH_ADDRESS_EXTRA = "SEARCH_TITLE_EXTRA";
 
```

Comment:
SEARCH_TYPE_EXTRA, RESULT_STREET_EXTRA, RESULT_LATLNG_EXTRA and SEARCH_ADDRESS_EXTRA should be private.